smokey/features/steps/client.py

```diff
@@ -0,0 +1,128 @@
+from behave import *
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Browsers(object):
+    """ Enum of the available browser configurations to test with """
+    # Local browsers
+    FIREFOX = 'Firefox'
+    CHROME = 'Chrome'
+
+    # Sauce Labs Desktop browsers
+    SAUCE_CHROME = 'Sauce-Chrome'
+    SAUCE_FIREFOX = 'Sauce-Firefox'
+    SAUCE_SAFARI = 'Sauce-Safari'
+    SAUCE_EDGE = 'Sauce-Edge'
+
+
+class Browser(object):
```

Comment:
As I mentioned in #2, I think at some point we're going to want to separate out the code that helps use Sauce from actual step definitions -- or at least make it clearer which step definitions use Sauce and which don't. For now, this is just fine.